vignettes/Introduction.Rmd

```diff
@@ -496,11 +516,67 @@ c2$v
 
 In the example `deep_clone` method above, we checked the name of each field to determine what to do with it, but we could also check the value, by using `inherits(value, "R6")`, or `is.environment()`, and so on.
 
+#### Cloning inheritance
```

Comment:
I'm not sure that this should be documented at this point -- it is how R6 currently behaves, but that actually feels like a bug to me. At any rate, if we do want to keep this behavior, it should be a separate PR.

Reply:
I have removed this subsection. Will make a different PR for that change.